crates/forge_domain/src/orch.rs

```diff
@@ -57,8 +57,10 @@ impl<A: App> Orchestrator<A> {
 
     async fn send_message(&self, agent: &Agent, message: ChatResponse) -> anyhow::Result<()> {
```

Comment:
There is `send` and `send_message` can you merge them in to `send`

Reply:
ok